ses-resources/helpers.sh

```diff
@@ -106,7 +106,8 @@ collect_info_from_ceph_cli() {
     {
         plugin_command "$ceph_shell $CEPH orch status" 2>&1
         plugin_command "$ceph_shell $CEPH cephadm get-ssh-config" 2>&1
-        plugin_command "$ceph_shell $CEPH orch ls --format yaml" 2>&1
+        plugin_command "$ceph_shell $CEPH orch ls --format yaml" 2>&1 |
+	    sed -e "/rgw_frontend_ssl_certificate: '/,/'/c \  rgw_frontend_ssl_certificate: '${CENSORED}'"
```

Comment:
```suggestion
	    awk -vf=1 "/BEGIN.*(CERTIFICATE|KEY)/{print ; print \"    ${CENSORED}\" ; f=0} /END.*(CERTIFICATE|KEY)/{f=1} f"
```

Reply:
How about this?  It gives the following output:

```
service_type: rgw
service_id: default.default
service_name: rgw.default.default
placement:
  hosts:
  - master
spec:
  rgw_frontend_port: 443
  rgw_frontend_ssl_certificate: '-----BEGIN CERTIFICATE-----
    <<CENSORED BY SUPPORTCONFIG PLUGIN>>
    -----END CERTIFICATE-----
    -----BEGIN PRIVATE KEY-----
    <<CENSORED BY SUPPORTCONFIG PLUGIN>>
    -----END PRIVATE KEY-----
    '
  ssl: true
```

Reply:
Thanks. I did it slightly different. Every line between '-----BEGIN ...' and '-----END ...' is replaced with CENSORED:
```
spec:
  rgw_frontend_ssl_certificate: '-----BEGIN PRIVATE KEY-----
    CENSORED
    CENSORED
    CENSORED
    CENSORED
    CENSORED
    -----END PRIVATE KEY-----
    -----BEGIN CERTIFICATE-----
    CENSORED
    CENSORED
    CENSORED
    CENSORED
    CENSORED
    -----END CERTIFICATE-----
    '
```

Reply:
Cool, LGTM!